### ISSUE TITLE
With 0.8.9 version, Index creation gets skip because of @Indexed field

### ISSUE THREAD

**Issue by shireeshshelke** [redis/redis-om-spring#396]
With `@Indexed` annotation the Index creation gets skipped with the following log message. 

`Skipping index creation for com.example.demo.PersonEntityIdx because On JSON, cannot set tag field to sortable - mobileNo` 

```
@Data
@AllArgsConstructor
@Document("person")
public class PersonEntity {

    @Id
    private String seqId;

    @Indexed
    private Long mobileNo;
}
```

However, when `@Searchable` annotation is applied on the mobileNo field, the index is successfully created.

**Comment by conifercone** [comment]
me too😑

**Comment by bsbodden** [comment]
Fixed in 0.9.0
